scripts/upload-docker-readme.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+
+# Copyright (c) 2024 The Jaeger Authors.
+# SPDX-License-Identifier: Apache-2.0
+set -euxf -o pipefail
+repo="$1"
+readme_path="$2"
+abs_readme_path=$(realpath "$readme_path")
+
+DOCKERHUB_TOKEN=${DOCKERHUB_TOKEN:-}
+dockerhub_repository="jaegertracing/$repo"
+dockerhub_url="https://hub.docker.com/v2/repositories/$dockerhub_repository/"
+
+if [ ! -f "$abs_readme_path" ]; then
+  echo "Warning: the dedicated README file for Docker image $repo was not found at path $abs_readme_path"
+  echo "It is recommended to have a dedicated README file for each Docker image"
+  exit 0
+fi
+
+readme_content=$(<"$abs_readme_path")
+
+set +x
+
+body=$(jq -n \
+  --arg full_desc "$readme_content" \
+  '{full_description: $full_desc}')
+
+dockerhub_response=$(curl -s -o /dev/null -w "%{http_code}" -X PATCH "$dockerhub_url" \
```

Comment:
> `-o /dev/null`

Will it not swallow the output / response, especially in case of an error?

Reply:
Yeah, makes sense.